tools/ci_build/get_docker_image.py

```diff
@@ -140,19 +74,33 @@ def main():
     if not use_container_registry:
         log.info("No container registry will be used")
 
-    tag = generate_tag(args.dockerfile, args.context, args.docker_build_args)
-
     full_image_name = (
-        "{}.azurecr.io/{}:{}".format(args.container_registry, args.repository, tag)
+        "{}.azurecr.io/{}:latest".format(args.container_registry, args.repository)
         if use_container_registry
-        else "{}:{}".format(args.repository, tag)
+        else "{}:latest".format(args.repository)
     )
 
     log.info("Image: {}".format(full_image_name))
 
-    if use_container_registry and container_registry_has_image(full_image_name, args.docker_path):
-        log.info("Pulling image...")
-        run(args.docker_path, "pull", full_image_name)
+    if use_container_registry:
+        run("az", "login", "--identity", "--username", "d1c41439-486c-4110-9052-d97199c8aa78")
```

Comment:
does this login command work when run locally? I got an error when I tried it. it would be nice if this script can run locally too.